docs/stable/getting_started/multi_machine_setup.md

```diff
@@ -0,0 +1,124 @@
+---
+sidebar_position: 3
+---
+
+# Multi-Machine Setup Guide
+
+This guide will help you get started with running ServerlessLLM on multiple machines by adding worker nodes on different machines, connecting them to the head node, and starting the `sllm-store-server` on the worker nodes. You can extend this setup to use as many nodes as you need. Please make sure you have installed the ServerlessLLM following the [installation guide](./installation.md) on all machines.
+
+## Multi-Machine Setup
+
+First, let's start a head node on the main machine and then add multiple worker nodes from other machines.
+
+### Step 1: Start the Head Node on the Main Machine
+
+1. **Activate the `sllm` environment and start the head node:**
+
+```bash
+conda activate sllm
+ray start --head --port=6379 --num-cpus=12 --num-gpus=0 \
+--resources='{"control_node": 1}' --block
+```
+
+### Step 2: Start Worker Nodes on Different Machines
+
+:::tip
+You can adjust the number of CPUs and GPUs based on the resources available on each machine.
+:::
+
+1. **On the first worker machine, activate the `sllm` environment and connect to the head node:**
+
+```bash
+conda activate sllm
+ray start --address=<HEAD_NODE_IP>:6379 --num-cpus=4 --num-gpus=2 \
+--resources='{"worker_node": 1, "worker_id_0": 1}' --block
+```
+
+2. **On the second worker machine, activate the `sllm` environment and connect to the head node:**
+
+```bash
+conda activate sllm
+ray start --address=<HEAD_NODE_IP>:6379 --num-cpus=4 --num-gpus=2 \
+--resources='{"worker_node": 1, "worker_id_1": 1}' --block
+```
+
+
+> Replace `<HEAD_NODE_IP>` with the actual IP address of the head node.
+
+You can continue adding more worker nodes by repeating the above steps on additional machines, specifying a unique `worker_id` for each node.
+
+### Step 3: Start ServerlessLLM Serve on the Head Node
+
+1. **On the head node machine, start ServerlessLLM Serve:**
+
+```bash
+conda activate sllm
+sllm-serve start
+```
+
+### Step 4: Start ServerlessLLM Store Server on the Worker Nodes
+
+1. **On each worker node machine, start the ServerlessLLM Store server:**
+
+```bash
+conda activate sllm
+sllm-store-server
+```
+
+### Step 5: Deploy a Model Using `sllm-cli`
+
+1. **Open a new terminal, activate the `sllm` environment, and set the `LLM_SERVER_URL` environment variable:**
```

Comment:
where? Since now we have multiple machines, it's important to tell users where to execute each command.

Reply:
Actually any machine is ok as we have to set the `LLM_SERVER_URL`, I would add this to doc.